rtengine/improccoordinator.cc

```diff
@@ -731,6 +731,17 @@ void ImProcCoordinator::updatePreviewImage(int todo, bool panningRelatedChange)
         // Remove transformation if unneeded
         bool needstransform = ipf.needsTransform(fw, fh, imgsrc->getRotateDegree(), imgsrc->getMetaData());
 
+		bool sharplablocal = false;//detect if Locallab sharpening is active
+        for (int sp = 0; sp < (int)params->locallab.spots.size(); sp++) {
+			if(params->locallab.spots.at(sp).expsharp) {
+				sharplablocal = true;
+			}
```

Comment:
Maybe a `break;` inside the `if` to not check remaining spots unnecessarily?
Or ditch `sharplablocal` entirely and
```
        for (int sp = 0; sp < (int)params->locallab.spots.size(); sp++) {
            if(params->locallab.spots.at(sp).expsharp) {
                params->dirpyrequalizer.cbdlMethod = "aft";
                break;
            }
        }
```